src/Microsoft.ApplicationInsights/TelemetryClient.cs

```diff
@@ -586,7 +586,11 @@ public void TrackRequest(RequestTelemetry request)
         /// </remarks>
         public void Flush()
         {
-            this.configuration.TelemetryChannel.Flush();
+            var telemetryChannel = this.configuration?.TelemetryChannel;
```

Comment:
Not sure what the convention of the SDK is, so you may dismiss this comment.
I was thinking, "var" should only be used in long LINQ statements or when the type is obvious from the same line, which is not the case here.

Reply:
Everything that not enforced is fine :-). We are using `var` widely in our repositories.

Reply:
Ok